packages/host/app/components/ai-assistant/apply-button/index.gts

```diff
@@ -34,6 +39,18 @@ const AiAssistantApplyButton: TemplateOnlyComponent<Signature> = <template>
         <CheckMark width='16' height='16' />
       {{else if (eq @state 'failed')}}
         <Exclamation width='16' height='16' />
+      {{else if (eq @state 'preparing')}}
+        <BoxelButton
+          @kind='secondary-dark'
+          @size='small'
+          class='apply-button'
+          tabindex='-1'
+          {{setCssVar boxel-button-text-color='var(--boxel-200)'}}
+          data-test-apply-state='preparing'
+          ...attributes
+        >
+          Working…
+        </BoxelButton>
```

Comment:
Should it be hoverable? It looks a bit weird:

![screencast 2025-03-20 09-11-55](https://github.com/user-attachments/assets/98d011bb-2155-468c-9986-c88d58774fc2)